### PR TITLE
fixes #99 `enablePjax` option for crud generator

### LINKED ISSUE #99: Use PJAX in GridView to replace ajax rendered container rather than submit whole page
Using PJAX consistently follows original implementation in Yii1 gridview is replaced via ajax callback. 
In Yii2-Gii, however this performance-wise feature has been replaced with submitting whole page.
Since few months ago Yii2 uses PJAX natively, generated Gridviews may also take advantage of it.

### PR DESCRIPTION
**Pull request by Faryshta**
Closes #99

### CODE REVIEW

File: generators/crud/Generator.php [hunk 1]
```diff
@@ -38,6 +38,11 @@ class Generator extends \yii\gii\Generator
     public $indexWidgetType = 'grid';
     public $searchModelClass = '';
 
+    /**
+     * @var bool if Pjax widget should be enabled.
```
Comment (SilverFire) [inline generators/crud/Generator.php]: `bool` -> `boolean`

Comment (SilverFire) [inline generators/crud/Generator.php]: I think it's better so change text to `whether to wrap GridView with Pjax`

File: generators/crud/Generator.php [hunk 2]
```diff
@@ -115,6 +121,7 @@ public function hints()
                 You may choose either <code>GridView</code> or <code>ListView</code>',
             'searchModelClass' => 'This is the name of the search model class to be generated. You should provide a fully
                 qualified namespaced class name, e.g., <code>app\models\PostSearch</code>.',
+            'enablePjax' => 'Enables the use of `yii\widgets\Pjax` inside the generated views.',
```
Comment (SilverFire) [inline generators/crud/Generator.php]: Backticks should be replaced with `<code>` tag

Comment (SilverFire) [inline generators/crud/Generator.php]: I'd like to suggest another text: `This indicates whether the generator should wrap the <code>GridView</code> widget on the index page with <code>Pjax</code> widget, that implements a jQuery plugin for website navigation by combining ajax and pushState. Set this to <code>true</code> if you want to get sorting, filtering and pagination without page refreshing.`

**Comment by samdark** [comment]
Looks fine.

**Comment by SilverFire** [comment]
Merged with minor adjustments https://github.com/yiisoft/yii2-gii/commit/177847f7b4ed07698a42d1e5f01bc792871210fb

**Comment by SilverFire** [comment]
Thank you!

**Pull request closed without merging**